### DISCUSSION TITLE
Load Metamask Api CDN to GrapesJS

### DISCUSSION THREAD

**Discussion by kmlcnclk** [GrapesJS/grapesjs#5391] [Q&A] [answered]
I want to add metamask api cdn to grapesjs but window.ethereum is returning undefined. How can we solve this problem. Can you help me?

**Comment by artf** [comment, marked as answer]
[This section](https://grapesjs.com/docs/modules/Components-js.html#basic-scripts) explains more in detail about using components with JS
